installer/install.php

```diff
@@ -186,7 +186,7 @@
 
         // Show success message if the installation is a complete success.
         if ($message === null) {
-            $absoluteURL = str_replace('/installer/install.php', '', $_SERVER['PHP_SELF']);
+            $absoluteURL = str_replace('/installer/install.php', '/', $_SERVER['PHP_SELF']);
```

Comment:
At this point in the installation, we have an absoluteURL in the session (as well as the database). I wonder, would it be more robust to use that here, rather than trying to interpret the absoluteURL from the $_SERVER['PHP_SELF']?

Reply:
Added another commit for that. Please check.